packages/greenbox-data/data/iotms.ts

```diff
@@ -1282,4 +1282,11 @@ export default [
     type: "item",
     item: "Sept-Ember Censer",
   }, // boxed Sept-Ember Censer
+  {
+    id: 11657,
```

Comment:
11658

Reply:
I'm curious, why is this the case? All the other iotms I looked at referred to the packaged item in their ID field.

Reply:
@gausie i think you're wrong on this one, all the other entries use the packaged item id

Reply:
Oh